pkg/apis/istio/v1alpha2/values_types.go

```diff
@@ -644,3 +646,19 @@ func FromInt(val int) IntOrStringForPB {
 func FromString(val string) IntOrStringForPB {
 	return IntOrStringForPB{intstr.FromString(val)}
 }
+
+// Validation checks
+func (t *PilotConfig) Validation(failOnMissingValidation bool, values *Values, icpls *IstioControlPlaneSpec) util.Errors {
```

Comment:
Maybe call this "Validate" instead? So we can write myPilot.Validate()

Reply:
renamed